src/routes/ColonyRoute.tsx

```diff
@@ -15,11 +18,13 @@ const ColonyRoute = () => (
       <ColonyDecisionProvider>
         <UserTokenBalanceProvider>
           <MemberModalProvider>
-            <UserTransactionContextProvider>
-              <ColonyLayout>
-                <Outlet />
-              </ColonyLayout>
-            </UserTransactionContextProvider>
+            <ColonyCreatedModalProvider>
```

Comment:
I feel this will get out of hand fast if we end up having a provider for every modal out there.

Reply:
Agreed